dataset/dictionary/characters.json5

```
@@ -358,43 +416,55 @@
   {
     en: "Boreas",
     ja: "ボレアス",
+    zh: “玻瑞亚斯”,
     tags: [ "mondstadt", "enemy-boss", "character-sub" ],
     notes: "日本語では単に「狼」と呼ばれることもあり、英語でも同様に \"wolf\" と呼ばれる",
```

I understand "Boreas" does not appear in the Chinese version. Then let's remove the `zh` property here. I also want to add an explanation about that.
(Translation for the note I added: "In the Chinese version, the name 'Boreas' does not appear in the game. It may be based on 'Lupus Boreas, Dominator of Wolves' in English version. (「北風の王狼、奔狼の領主」 in Japanese)"
```suggestion
    tags: [ "mondstadt", "enemy-boss", "character-sub" ],
    notes: "日本語では単に「狼」と呼ばれることもあり、英語でも同様に \"wolf\" と呼ばれる。中国語版では「ボレアス」という名前はゲーム中に登場しない。英語版の \"Lupus Boreas, Dominator of Wolves\" (日本語では「北風の王狼、奔狼の領主」) を元にしていると思われる。",
```